doc/source/conf.py

```diff
@@ -166,6 +166,7 @@
         ):
     warnings.filterwarnings(
         'once', message='.*' + key)
+warnings.filterwarnings("ignore", category=DeprecationWarning)
```

Comment:
Is this needed? Typically we add deprecations here: https://github.com/scipy/scipy/blob/aea3e2a8ef68713eb42d77ce00da2d98e34cc763/doc/source/conf.py#L316-L326

Reply:
Indeed, nice catch. That's the result of an accidental `commit -a`: I couldn't figure out how to remove the sphinx elevate-warnings-to-errors option with the `dev.py doc` interface, so I had to modify the config file. Will back it out...